metafacture-yaml/src/test/java/org/metafacture/yaml/YamlEncoderTest.java

```diff
@@ -277,8 +277,6 @@ public void testShouldPrefixOutputWithNewline() {
     public void testShouldPrefixPrettyPrintedOutputWithNewline() {
```

Comment:
This test is now identical to the previous one (`testShouldPrefixOutputWithNewline`) and serves no purpose. Either leave it untouched (to show that `setPrettyPrinting()` is a noop) or remove it entirely.

Reply:
Yeah good point. Gonna remove it.